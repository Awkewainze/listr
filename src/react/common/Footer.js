import React from 'react';
import './Footer.scss';

const Footer = () => (
  <footer className="Footer">
    <small className="Attribution">
      Logo made by <a href="https://www.flaticon.com/authors/vectors-market" className="Attribution__link" title="Vectors Market">Vectors Market</a> from <a href="https://www.flaticon.com/" className="Attribution__link" title="Flaticon"> www.flaticon.com</a>
    </small>
  </footer>
);

export default Footer;