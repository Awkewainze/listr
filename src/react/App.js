import React from 'react';
import './App.scss';
import Header from './common/Header';
import Footer from './common/Footer';
import { Switch, Route } from 'react-router-dom';
import ActiveListContainer from './list/ActiveListContainer';

function App() {
  return (
    <div className="App">
      <Header/>
      <Switch>
        <Route path="/" component={ActiveListContainer} />
      </Switch>
      <Footer/>
    </div>
  );
}

export default App;
