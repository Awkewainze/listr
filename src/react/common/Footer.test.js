import React from 'react';
import { render } from '@testing-library/react';
import Footer from './Footer';

it('matches snapshot', () => {
  const { container } = render(<Footer />);
  expect(container.firstChild).toMatchSnapshot();
});