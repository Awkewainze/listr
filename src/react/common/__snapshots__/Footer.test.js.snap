// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<footer
  class="Footer"
>
  <small
    class="Attribution"
  >
    Logo made by 
    <a
      class="Attribution__link"
      href="https://www.flaticon.com/authors/vectors-market"
      title="Vectors Market"
    >
      Vectors Market
    </a>
     from 
    <a
      class="Attribution__link"
      href="https://www.flaticon.com/"
      title="Flaticon"
    >
       www.flaticon.com
    </a>
  </small>
</footer>
`;
